modules/qna/src/views/full/Components/QnA.tsx

```diff
@@ -268,6 +326,26 @@ const QnA: FC<Props> = props => {
           )}
         </div>
       )}
+
+      {showContentForm && (
+        <ContentAnswerForm
+          bp={bp}
+          deleteContent={() => deleteContentAnswer()}
+          editingContent={editingContent.current}
+          formData={contentAnswers[editingContent.current]}
+          onUpdate={data => updateContentAnswers(data)}
+          close={closingKey => {
+            setTimeout(() => {
+              if (closingKey === editingContent.current) {
+                editingContent.current = null
+                setShowContentForm(false)
+              } else {
+                setForceUpdate(!forceUpdate)
```

Comment:
I'm curious when this scenario happens?

Reply:
When you click on another content, it's just to update the form content without closing/reopening the side drawer